cmd/utils.go

```diff
@@ -284,19 +295,24 @@ func getProjectConfig(config *RootCommandConfig) (ProjectConfig, error) {
 		Debug.log("Project config file set to: ", appsodyConfig)
 		err := v.ReadInConfig()
 		if err != nil {
-			var tempProjectConfig ProjectConfig
-			return tempProjectConfig, errors.Errorf("Error reading project config %v", err)
+			return projectConfig, errors.Errorf("Error reading project config %v", err)
+
+		}
+		err = v.Unmarshal(&projectConfig)
+		if err != nil {
+			return projectConfig, errors.Errorf("Error reading project config %v", err)
 
 		}
-		stack := v.GetString("stack")
-		Debug.log("Project stack from config file: ", stack)
+
+		Debug.log("Project stack from config file: ", projectConfig.Stack)
 		imageRepo := config.CliConfig.GetString("images")
 		Debug.log("Image repository set to: ", imageRepo)
 		if imageRepo != "index.docker.io" {
-			stack = imageRepo + "/" + stack
+			projectConfig.Stack = imageRepo + "/" + projectConfig.Stack
 		}
-		Debug.log("Pulling stack image as: ", stack)
-		config.ProjectConfig = &ProjectConfig{stack}
+		Debug.log("Pulling stack image as: ", projectConfig.Stack)
```

Comment:
I see the message was there before, but it doesn't look like we are pulling the stack image yet. I think we sould delete this log message as we print out the stack in several other debug logs.

Reply:
I agree